tony-core/src/main/java/com/linkedin/tony/Constants.java

```diff
@@ -12,6 +12,7 @@
 public class Constants {
   // File Permission
   public static final FsPermission perm770 = new FsPermission((short) 0770);
+  public static final FsPermission perm777 = new FsPermission((short) 0777);
```

Comment:
The naming convention for constants is ALL_CAPS

Reply:
Updated.